docs/docsite/rst/shared_snippets/installing_collections_git_repo.txt

```diff
@@ -0,0 +1,83 @@
+Installing a collection from a git repository
+---------------------------------------------
+
+You can install a collection in a git repository by providing the URI to the repository instead of a collection name or path to a ``tar.gz`` file. The URI should be prefixed with ``git+`` (or with ``git@`` to use a private repository with ssh authentication) and optionally supports a comma separated version (for example, a commit or tag).
+
+.. code-block:: bash
+
+   # Install a collection in a repository using the latest commit on the branch 'devel'
+   ansible-galaxy collection install git+https://github.com/organization/repo_name.git,devel
+
+   # Install a collection from a private github repository
+   ansible-galaxy collection install git@github.com:organization/repo_name.git
+
+   # Install a collection from a local git repository
+   ansible-galaxy collection install git+file:///home/user/path/to/repo/.git
+
+In a ``requirements.yml`` file, you can also use the ``scm`` and ``version`` keys in addition to using the ``git+repo,version`` syntax for the collection name.
```

Comment:
We should probably update https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/shared_snippets/installing_multiple_collections.txt to list the keys that are supported in a collection requirements and what they can be set to.

Reply:
Updated in d841de9c69f24d8fc433e6787c7071e68b7f28a2